src/MongoDB/Manager.c

```diff
@@ -58,8 +58,12 @@ PHP_METHOD(Manager, __construct)
 {
 	php_phongo_manager_t     *intern;
 	zend_error_handling       error_handling;
+#if PHP_VERSION_ID >= 70000
+        zend_string              *uri_string;
+#else
```

Comment:
I don't think this was needed? the zend_parse_parameter() with 's' looks like would be fine as we are still passing char* to phongo_manager_init(), which this change does anyway too..